docs/api-reference.rst

```diff
@@ -1,14 +1,16 @@
 .. _api-reference:
 
 =============
-API Reference
+API reference
```

Comment:
I'm not sure if that's new or not, but we got all the `bytes` / `str` methods documented for page input classes; I don't think it's intended. That's probably unrelated to this PR.

Reply:
Not what I’m getting:

![Screenshot_20230330_093847](https://user-images.githubusercontent.com/705211/228764326-094ab733-1095-4cac-8038-cf6f63d46d8b.png)
![Screenshot_20230330_093923](https://user-images.githubusercontent.com/705211/228764466-91a36cae-a4fd-4623-a5e2-fb7bf4b898e1.png)

Reply:
I’ve tried recreating the `docs` Tox environment, and I still get the expected rendering. Could you try recreating your `docs` Tox environment on your end?

Reply:
It turns out I was not using the right docs environment. I've installed docs/requirements.txt, and the problem went away. Old Sphinx, probably.